packages/edit-post/src/components/header/index.js

```diff
@@ -6,12 +6,11 @@ import { Button } from '@wordpress/components';
 import { PostPreviewButton, PostSavedState } from '@wordpress/editor';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { cog } from '@wordpress/icons';
-import { PinnedItems } from '@wordpress/interface';
+import { PinnedItems, AdminMenuToggle } from '@wordpress/interface';
```

Comment:
I know it's not a component, but the _is_ an existing thing that collapses the admin menu. I'm not sure what it's called, but it's at the bottom of the menu. I think using the name `AdminMenuToggle` could be a bit confusing since one already exists. I'm not sure why the existing one isn't used for this fullscreen thing.

Reply:
I guess I think of the `#collapse-button` option that exists at the bottom of the menu as a different feature; The collapse button changes the width of the admin sidebar, not the visibility.

This new `AdminMenuToggle` component toggles the visibility.

The two features can work together; You can collapse the admin sidebar _and_ toggle its visibility.